_drgn.pyi

```diff
@@ -200,6 +201,15 @@ class Program:
             the given name
         """
         ...
+    def symbols(self, name: str) -> List[Symbol]:
+        """
+        Get a list of global and local symbols which have the given name.
+
+        :param name: The symbol name.
+        :raises LookupError: if no symbol contains the given address or matches
```

Comment:
If it were a blank slate design I'd pick an empty list as a return value (since `[]` is definitely a valid `List[Symbol]`). But given that symbol() already raises an exception when it doesn't find anything, I like sticking with the exception as written. That way there are fewer surprises going between APIs.